src/core/ee/emotion.hpp

```diff
@@ -48,8 +50,9 @@ class EmotionEngine
         void handle_exception(uint32_t new_addr, uint8_t code);
         void deci2call(uint32_t func, uint32_t param);
     public:
-        EmotionEngine(Cop0* cp0, Cop1* fpu, Emulator* e, uint8_t* sp, VectorUnit* vu0, VectorUnit* vu1);
+        EmotionEngine(Cop0* cp0, Cop1* fpu, Emulator* e, uint8_t* sp, VectorUnit* vu0, VectorUnit* vu1, EEBreakpointList* ee_breakpoints);
         static const char* REG(int id);
+        static const char* COP0_REG(int id);
```

Comment:
Recommend marking function as `const`.

```suggestion
        static const char* COP0_REG(int id) const;
```